internal/test/wsu/wsu_test.go

```diff
@@ -87,11 +87,16 @@ func createHostFile(vmList []e2ef.TestWindowsVM) (string, error) {
 	}
 	defer hostFile.Close()
 
+	// Give a loop back ip as internal ip, this would never show up as
+	// private ip for any cloud provider. This is a dummy for testing purposes
+	loopbackIP := "127.0.0.1"
+
```

Comment:
I am fine with this hack. Just add a comment saying it is a hack to avoid changes  to the `Credentials` interface or making AWS API calls at this juncture and it would need to be fixed if we ever want to add Azure e2e tests.

Reply:
Added